pkg/cmd/grafana-cli/commands/migrate_database_command.go

```diff
@@ -0,0 +1,7 @@
+package commands
+
+// migrateDatabaseCommand is just a stub that does nothing. Migrations are run as part of every database command
+// and so we just need that and this can be empty.
+func migrateDatabaseCommand(c CommandLine) error {
```

Comment:
Added a command to just run migration so it is easier to test the migration. Mainly just makes the log output better than running server and finding the logs there.

Reply:
Great! Wish this would have been a separate PR :) but it doesn't matter now.